src/cliffordsynthesis/encoding/TwoQubitEncoder.cpp

```diff
@@ -0,0 +1,251 @@
+//
+// Created by Velsh Aleksei on 16.06.23.
+//
+
+#include "cliffordsynthesis/encoding/TwoQubitEncoder.hpp"
+
+#include "LogicTerm/LogicTerm.hpp"
+#include "operations/OpType.hpp"
+#include "utils/logging.hpp"
+
+namespace cs::encoding {
+
+using namespace logicbase;
+
+void encoding::TwoQubitEncoder::collectTwoQubitGateVariables(
```

Comment:
This method can be simplified by calling the method from the base class which already adds all variables except the `twoQubitGates[qubit][qubit]` one